### PR TITLE
Add support for optional monthStr props

### PR DESCRIPTION
**Pull request by bseib**
You may pass an array of 12 strings to be used when rendering the month
in the Datepicker, DateRangePicker, or DatetimePicker.

The default is:

    monthStr = ['01', '02', '03', '04', '05', '06',
                '07', '08', '09', '10', '11', '12']

### CODE REVIEW

File: README-CN.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ const defaultPopperProps = {
 | Name                | Type         | DefaultValue                                 | Description  |
 | ------------------- | ------------ | -------------------------------------------- | ------------ |
 | `dayStr`            | `Array`      | `['日', '一', '二', '三', '四', '五', '六']`   | Used to set day name |
+| `monthStr`          | `Array`      | `['01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12']`   | Used to set day name |
```
Comment (livelybone) [inline README-CN.md]: Used to set month name

File: README-CN.md [hunk 2]
```diff
@@ -106,6 +107,7 @@ const defaultPopperProps = {
 | Name              | Type                   | DefaultValue                                 | Description  |
 | ----------------- | ---------------------- | -------------------------------------------- | ------------ |
 | `dayStr`          | `Array`                | `['日', '一', '二', '三', '四', '五', '六']`   | Used to set day name |
+| `monthStr`        | `Array`                | `['01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12']`   | Used to set day name |
```
Comment (livelybone) [inline README-CN.md]: Used to set month name

File: README-CN.md [hunk 3]
```diff
@@ -116,6 +118,7 @@ const defaultPopperProps = {
 | --------------------- | ---------------------- | -------------------------------------------- | ------------ |
 | `value`               | `Array`                | `[null, null]`                               | Used to set day name |
 | `dayStr`              | `Array`                | `['日', '一', '二', '三', '四', '五', '六']`   | Used to set day name |
+| `monthStr`            | `Array`                | `['01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12']`   | Used to set day name |
```
Comment (livelybone) [inline README-CN.md]: Used to set month name

File: README-CN.md (hunk 1 above)
Comment (bseib) [reply README-CN.md]: Whoops! Copy/paste typos in several places! 🤦 I'll push an updated branch. 😃

**Comment by bseib** [comment]
Typos for day -> month have been corrected.

**Comment by livelybone** [comment]
OK, I notice that the month selection panel did not show as the monthStr you set when I run your branch month-strings. This seems be unexpected and I think you should fix this bug. The step for repetition:
1. run command `npm run dev`
2. open `http://localhost:3000/examples/test.html` in browser
3. click one datepicker
4. click the month, and then shows the month selection panel

**Comment by bseib** [comment]
Ah, I didn't see that month picker too. I updated some code, see what you think. 😄

**Comment by bseib** [comment]
@livelybone was there a problem with this code? Any reason not to merge?

**Comment by livelybone** [comment]
It's a little mistake

**Pull request merged**